.github/CODEOWNERS

```diff
@@ -99,6 +99,9 @@
 
 # Blog project
 /core/domain/blog*.py @iamprayush
+/core/controllers/blog_dashboard*.py @iamprayush
+/core/controllers/blog_admin*.py @iamprayush
```

Comment:
Should we move this to the restricted page section?

Reply:
Moved blog admin page to restricted pages.